test/hw7/test_task1_recdict.py

```diff
@@ -0,0 +1,42 @@
+from typing import NoReturn, Optional
+
+import pytest
+
+from homework.hw7.task1_recdict import find_occurrences
+
+
+example_tree = {
```

Comment:
Use resource files for JSON like objects. It's a bad practice to use them in .py files. Also in tests for that purpose we can use fixtures